mysite/lti/admin.py

```diff
@@ -1,10 +1,28 @@
 from django.contrib import admin
+from django.forms.models import ModelForm
 
-from lti.models import LTIUser, CourseRef
+from lti.models import LTIUser, CourseRef, LtiConsumer
+
+
+class LtiConsumerForm(ModelForm):
+    def __init__(self, *args, **kwargs):
+        super(LtiConsumerForm, self).__init__(*args, **kwargs)
+
+        class Meta:
+            model = LtiConsumerAdmin
+
+    def clean(self):
+        cleaned_data = self.cleaned_data
+        instance_guid = cleaned_data.get('instance_guid')
+
+        if not instance_guid:
+            cleaned_data['instance_guid'] = None
```

Comment:
Looks like `instance_guid` is set to `null=True, blank=True`. Is there any specific reason for using this clean mehtod here.?